libs/blocks/quiz-results/quiz-results.css

```diff
@@ -0,0 +1,59 @@
+@import url("../section-metadata/section-metadata.css");
+
+.quiz-results {
+  visibility: hidden;
+}
+
+.quiz-results.show {
+  visibility: unset;
+}
+
+.quiz-results .tablet-portrait-up, .quiz-results .tablet-landscape-up,
+.quiz-results .section.desktop-up, .quiz-results .big-desktop-up {
```

Comment:
nit. New line after the comma

Reply:
Commas should be in the appropriate places in CSS now.